README.md

```diff
@@ -1,58 +1,95 @@
-# Sonarr
+<p align="center">
+<img src="./Logo/256.png" alt="Sonarr">
+</p>
 
 Sonarr is a PVR for Usenet and BitTorrent users. It can monitor multiple RSS feeds for new episodes of your favorite shows and will grab, sort and rename them. It can also be configured to automatically upgrade the quality of files already downloaded when a better quality format becomes available.
 
-## Major Features Include:
+## Getting Started
 
-* Support for major platforms: Windows, Linux, macOS, Raspberry Pi, etc.
-* Automatically detects new episodes
-* Can scan your existing library and download any missing episodes
-* Can watch for better quality of the episodes you already have and do an automatic upgrade. *eg. from DVD to Blu-Ray*
-* Automatic failed download handling will try another release if one fails
-* Manual search so you can pick any release or to see why a release was not downloaded automatically
-* Fully configurable episode renaming
-* Full integration with SABnzbd and NZBGet
-* Full integration with Kodi, Plex (notification, library update, metadata)
-* Full support for specials and multi-episode releases
-* And a beautiful UI
+[![Installation](https://img.shields.io/badge/wiki-installation-brightgreen.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/Installation)
+[![Docker](https://img.shields.io/badge/wiki-docker-1488C6.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
+[![FAQ](https://img.shields.io/badge/wiki-FAQ-BF55EC.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/FAQ)
+
+- [Install Sonarr for your desired OS](https://github.com/Sonarr/Sonarr/wiki/Installation) or use [Docker](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
+- Connect to the UI through http://localhost:8989 or http://your-ip:8989 in your web browser
+- See the [Wiki](https://github.com/Sonarr/Sonarr/wiki) for further configuration
+
+## Downloads
+
+Type | develop (stable) |
```

Comment:
Develop and stable together doesn’t  really make sense to me. Releases on the release page are from master, not develop.